lms/validation/authentication/_bearer_token.py

```diff
@@ -109,69 +84,32 @@ def lti_user(self, location):
           ``"Bearer <ENCODED_JWT>"`` format.
         :raise ValidationError: if the JWT's payload is invalid, for example if
           it's missing a required parameter
-
-        :rtype: LTIUser
         """
-        try:
-            return self.parse(location=location)
```

Comment:
I found this a bit difficult to follow, it does what's expected in marshmallow but I reckon this version (copied from the Oauth equivalent) is more explicit.

In the older version this would rely on `@marshmallow.pre_load` to get the authorization header and then marshmallow will iterate over every field declared in the schema to get the value as part of `data` and finally ```@marshmallow.post_load``` would create the LTIUser.